<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorSLAM</class>
 <widget class="QWidget" name="InspectorSLAM">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>SLAM</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2"/>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_pVolumeSizeLE">
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Volume Resolution</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Truncated Distance</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Volume Size</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Max Integration Weight</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="m_pTruncatedDistanceLE">
       <property name="text">
        <string>30</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="m_pMaxIntegrationWeightLE">
       <property name="text">
        <string>200</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Depth Threshold</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="m_pMinDepthLE">
         <property name="text">
          <string>500</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="m_pMaxDepthLE">
         <property name="text">
          <string>8000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="m_pResolutionTL">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="m_pResolutionHS">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="m_pReconstructButton">
     <property name="text">
      <string>Reset Reconstruction</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_pReconstructButton</sender>
   <signal>clicked()</signal>
   <receiver>InspectorSLAM</receiver>
   <slot>valueChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pResolutionHS</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>InspectorSLAM</receiver>
   <slot>sliderValueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>valueChanged()</slot>
  <slot>sliderValueChanged(int)</slot>
 </slots>
</ui>
